specification/datashare/resource-manager/readme.python.md

```diff
@@ -0,0 +1,25 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+```yaml $(python)
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: Microsoft.DataShare
```

Comment:
this namespace should be consistent with readme.az.md in the format of azure.mgmt.service